great_expectations/data_context/templates.py

```diff
@@ -79,55 +80,20 @@
   #     file_extension: json
   #     filepath_template: '{4}/{0}/{1}/{2}/{3}.{file_extension}'
 
-  # FIXME: These configs are temporarily commented out to facititate refactoring Stores.
-
-  # local_profiling_store:
-  #   module_name: great_expectations.data_context.store
-  #   class_name: FilesystemStore
-  #   store_config:
-  #     base_directory: uncommitted/profiling/
-  #     serialization_type: json
-  #     file_extension: .json
-
-  # local_workbench_site_store:
-  #   module_name: great_expectations.data_context.store
-  #   class_name: FilesystemStore
-  #   store_config:
-  #     base_directory: uncommitted/documentation/local_site
-  #     file_extension: .html
-
-  # shared_team_site_store:
-  #   module_name: great_expectations.data_context.store
-  #   class_name: FilesystemStore
-  #   store_config:
-  #     base_directory: uncommitted/documentation/team_site
-  #     file_extension: .html
-
-  # fixture_validation_results_store:
-  #   module_name: great_expectations.data_context.store
-  #   class_name: FilesystemStore
-  #   store_config:
-  #     base_directory: fixtures/validations
-  #     file_extension: .zzz
 
+  evaluation_parameter_store:
+    # Evaluation Parameters enable dynamic expectations. Read more here:
+    # https://docs.greatexpectations.io/en/v0.8.x/reference/evaluation_parameters.html
```

Comment:
Let's get the version dynamically - see an example in cli/datasource.py - add_datasource method

Reply:
Oooh GREAT catch. Thanks for the pointer!